### DISCUSSION TITLE
PP calculation not calculating properly.

### DISCUSSION THREAD

**Discussion by xDololow** [ppy/osu#15367] [Q&A] [answered]
Here's a video of this issue: https://puu.sh/IlE3q.mp4
PP suddenly just goes from 500 to 650 after the first object clicked after the break.

**Comment by peppy** [comment]
I believe this may just be limitations of the current calculation method. Are you able to reproduce on other beatmaps which aren't over 10 stars? Does it always happen after long breaks?

**Reply by xDololow** [reply]
> Are you able to reproduce on other beatmaps which aren't over 10 stars? 

I tried a few maps and they doesn't seem to have this issue. 

> Does it always happen after long breaks?

No, one map has a spinner at the beginning was giving 35pp at the beginning of it and 75pp at the end of the spinner. (https://osu.ppy.sh/beatmapsets/182815#osu/438187 this map with DT which is almost 10*)

PP calculator in lazer works different from the "unofficial" ones.

**Reply by peppy** [reply]
Of course the calculation will be different from other displays (they are using different code). There are known shortcomings which come from the way lazer is processing difficulty. @smoogipoo may be able to give you a better answer on this front (whether the issue you report is a localised issue or a larger one that may take time to fix).

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15408
